airflow/api/common/experimental/mark_tasks.py

```diff
@@ -49,6 +49,72 @@ def _create_dagruns(dag, execution_dates, state, run_id_template):
     return drs
 
 
+def _verify_tree(dag, dates, state):
```

Comment:
Can we name this something that indicates it does an action. Something along the lines of `_correct_tree` would be great because when I first read this I assume it would return a bool and take no action.